persistence-cassandra/javadsl/src/main/scala/com/lightbend/lagom/internal/javadsl/persistence/cassandra/CassandraReadSideHandler.scala

```diff
@@ -21,7 +21,7 @@ import com.lightbend.lagom.javadsl.persistence.ReadSideProcessor.ReadSideHandler
 import com.lightbend.lagom.javadsl.persistence.cassandra.CassandraSession
 import com.lightbend.lagom.javadsl.persistence.AggregateEvent
 import com.lightbend.lagom.javadsl.persistence.AggregateEventTag
-import com.lightbend.lagom.javadsl.persistence.Offset
+import com.lightbend.lagom.javadsl.persistence.{ Offset => LagomOffset }
```

Comment:
I was getting a bit tired of so many `Offset` types in the code and having to scroll to the imports every time to see what `Offset` was the code referring to (Lagom's or Akka Persistence Query) so I started using aliases anywhere I would step into. 

This change makes the PR bigger than it should but hopefully also makes the PR more readable.

Reply:
This is really good. Thanks for doing that. It's making everything much easier to read and helps to review this PR.